src/bitcoin-tx.cpp

```diff
@@ -772,9 +772,7 @@ static std::string readStdin()
     if (ferror(stdin))
         throw std::runtime_error("error reading stdin");
 
-    boost::algorithm::trim_right(ret);
-
-    return ret;
+    return TrimString(ret);
```

Comment:
This SHOULD be a locale trim!

Reply:
why?

Reply:
It's interacting with the CLI, which is expected to behave in the user's configured locale.

Reply:
When processing text, or other locale-formatted things like dates/..., I can see that. But this is just accepting hex-encoded tx data?